docs/site/custom-commands.md

```diff
@@ -96,3 +96,45 @@ There are three ways to bind shortcuts to custom commands:
     * One of `tab`, `backspace`, `,`, `.`, or `-` 
 2. Bind `calva.runCustomREPLCommand` to a shortcut with whatever code you want to evaluate in `args` key. You have access to the substitution variables here as well.
 3. Bind `calva.runCustomREPLCommand` to a keyboard shortcut referencing the `key` of one of your `calva.customREPLCommandSnippets`. (If not using any of the `key`s mentioned in **1.**)
+
+## Custom REPL hover snippets
+
+Calva supports custom snippets that will display their result inside the tooltip. They will only work when connected to a repl, since they eval code in it.
+This is mostly useful for tooling authors that want to integrate with calva.
+Be careful with these, since they will be executed anytime Calva displays a tooltip. So they should be fast and probably not have any sideeffects.
```

Comment:
```suggestion
Be careful with these, since they will be executed anytime Calva displays a tooltip. So they should be fast and probably not have any side effects.
```